### ISSUE TITLE
1.7.1 : bundle install failing on OSX with "requires ruby version >= 2.1.0" and 2.1.8 is current

### ISSUE THREAD

**Issue by cdenneen** [sparklemotion/nokogiri#1627]
**What is the output of `gem install`?**
```
# gem install nokogiri
Fetching: nokogiri-1.7.1.gem (100%)
Building native extensions.  This could take a while...
Successfully installed nokogiri-1.7.1
1 gem installed
```
```
# ruby -v
ruby 2.1.8p440 (2015-12-16 revision 53160) [x86_64-darwin15.0]
```
```
# bundle install
Installing nokogiri 1.7.1 with native extensions

Gem::InstallError: nokogiri requires Ruby version >= 2.1.0.
```

**What operating system are you using?**
OSX 10.11.6

```
# bundle config
Settings are listed in order of priority. The top value will be used.
jobs
Set via BUNDLE_JOBS: "8"
Set for the current user (/Users/cdenneen/.bundle/config): "2"

path
Set for the current user (/Users/cdenneen/.bundle/config): ".bundle"

disable_shared_gems
Set for the current user (/Users/cdenneen/.bundle/config): "true"

gem.test
Set for the current user (/Users/cdenneen/.bundle/config): "false"

gem.mit
Set for the current user (/Users/cdenneen/.bundle/config): "false"

gem.coc
Set for the current user (/Users/cdenneen/.bundle/config): "false"

without
Set for the current user (/Users/cdenneen/.bundle/config): "development:system_tests"

build.nokogiri
Set for the current user (/Users/cdenneen/.bundle/config): "--use-system-libraries"

specific_platform
Set for the current user (/Users/cdenneen/.bundle/config): "true"
```

**Comment by larskanis** [comment]
The bundle command probably runs with a different ruby version. You may re-install bundler for the current ruby version per `gem inst bundler` or start bundler per `ruby -S bundle install` .

**Comment by cdenneen** [comment]
@larskanis reinstalling bundler doesn't fix the problem. 
Oddly my path had the rbenv shims before /usr/local/bin but for some reason which bundler was using that one which was from shipped ruby 2.0.0.
